docs/celo-codebase/protocol/bridging/optics-add-new-token.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Optics Add New Token
+description: How to add a new token to Optics v2.
+---
+
+import YouTube from '@components/YouTube';
+import ImageWrapper from '@components/ImageWrapper'
+
+# Add New Token to Optics
+
+How to add a new token to Optics v2.
+
+---
+
+## Getting Started
+
+Optics is a cross-chain communication system designed for use on any EVM chain. The Token Bridge xApp (an application built on top of Optics) facilitates the transfer of tokens between chains. Adding a new token to the Optics Bridge can be done at any time using the [
```

Comment:
minor nit, update the text to say "Adding a new token to the Optics Bridge UI can be done at any..." to remove ambiguity about whether this tutorial is for creating new token contracts or just updating the UI. wdyt?